backend/apps/accounts/views.py

```
@@ -429,11 +33,13 @@ def post(self, request):
             )
             reset_token = PasswordResetToken.objects.create(user=user)
 
+            # Build reset link
             reset_url = frontend_url(
```

Ruff (0.15.20)

[error] Undefined name `frontend_url`

(F821)